rai/node/cli.cpp

```diff
@@ -433,7 +433,14 @@ std::error_code rai::handle_node_options (boost::program_options::variables_map
 							rai::raw_key key;
 							auto error (existing->second->store.fetch (transaction, account, key));
 							assert (!error);
-							std::cout << boost::str (boost::format ("Pub: %1% Prv: %2%\n") % account.to_account () % key.data.to_string ());
+							if (rai::pub_key (key.data) == account)
+							{
+								std::cout << boost::str (boost::format ("Pub: %1% Prv: %2%\n") % account.to_account () % key.data.to_string ());
+							}
+							else
+							{
+								std::cout << boost::str (boost::format ("Pub: %1% Prv: %2% Invalid private key!\n") % account.to_account () % key.data.to_string ());
```

Comment:
Should this be stderr, rather than stdout ?

Reply:
If add "Invalid private key" to different line